### PR TITLE
Update contact info

### PR DESCRIPTION
**Pull request by bvhme**
@ElenaFdR what do you think of this?

### CODE REVIEW

File: _includes/footer.liquid [hunk 1]
```diff
@@ -22,9 +22,17 @@
     <p>More on <a href="https://projects.publiccode.net">projects.publiccode.net</a></p>
   </section>
   <section>
-    <h3>Contact</h3>
+    <h3><a href="https://about.publiccode.net/organization/contact-details.html">Contact</a></h3>
     <ul>
-      <li><a href="mailto:info@publiccode.net">info@publiccode.net</a></li>
+      <li><a href="mailto:info@publiccode.net">Email: info@publiccode.net</a></li>
```
Comment (bvhme) [inline _includes/footer.liquid]:
```suggestion
      <li>Email: <a href="mailto:info@publiccode.net">info@publiccode.net</a></li>
```

**Comment by bvhme** [comment]
Points:

1. Fixed
2. This text isn't necessarily in Italics but in the HTML semantic `<address>` tag (in order to comply with the standard), your browser renders this in italic since a style override lacks in our design. I've made https://github.com/publiccodenet/jekyll-theme/issues/19
3. Tiny font might be an idea. Technically and on smaller screens this content is at the very bottom and not part of the 'about us' section, but that is where it renders on larger screens. Another solution might be a header for this section.
4. Yes, across some of them I'm guessing, perhaps all.

I'll continue thinking about this soon.

**Comment by ElenaFdR** [comment]
Looking at these changes with fresh eyes, I think they're useful enough to be published now. We can always change them again in the future.
(I've also added the mailing list call to action to this footer change.)

**Comment by ElenaFdR** [comment]
Looking at these changes with fresh eyes, I think they're useful enough to be published now. We can always change them again in the future.
(I've also added the mailing list call to action to this footer change, as part of [about repo #318](https://github.com/publiccodenet/about/issues/318)).

**Pull request merged**